src/components/CommentReply/style.scss

```diff
@@ -1,14 +1,14 @@
 .comment-reply {
-  margin-top: 20px;
-  border-left: 5px solid $color-textarea-background;
-  padding-left: 7px;
+  margin-top: 0px;
+  padding: 0px 20px 10px 20px;
   pointer-events: auto;
+  position: relative;
 
   &__text {
     color: $color-white;
     font-size: 1em;
     font-weight: bold;
-    margin-top: 12px;
+    margin-top: 0px;
```

Comment:
Very much nitpick – our CSS linting normally enforces writing 0 values without units for readability.